modules/plugin/geopkg/src/main/java/org/geotools/geopkg/GeoPkgDialect.java

```diff
@@ -496,7 +496,9 @@ public void setValue(
                 ps.setString(column, ((Time) value).toString());
                 break;
             case Types.TIMESTAMP:
-                ps.setString(column, ((Timestamp) value).toString());
+                //2020-02-19 23:00:00.0  --> 2020-02-19 23:00:00.0Z
+                //We need the "Z" or sql lite will interpret the value as local time
+                ps.setString(column, ((Timestamp) value).toString()+"Z");
```

Comment:
That is part of the actual geopackage spec right?

Reply:
no - this is how the "datetime()" function in sql lite works.  
Its a bit of black magic...